src/explore-education-statistics-admin/src/pages/legacy-releases/LegacyReleasesPageContainer.tsx

```diff
@@ -3,17 +3,12 @@ import { PublicationContextProvider } from '@admin/contexts/PublicationContext';
 import {
   legacyReleaseCreateRoute,
   legacyReleaseEditRoute,
-  legacyReleasesIndexRoute,
 } from '@admin/routes/legacyReleaseRoutes';
 import { PublicationRouteParams } from '@admin/routes/routes';
 import React from 'react';
 import { RouteComponentProps, Switch } from 'react-router';
 
-const routes = [
-  legacyReleasesIndexRoute,
-  legacyReleaseCreateRoute,
-  legacyReleaseEditRoute,
-];
+const routes = [legacyReleaseCreateRoute, legacyReleaseEditRoute];
```

Comment:
Do we actually still need this Component if we no longer have a stand-alone Legacy Releases set of pages?

Reply:
The create and edit pages are still stand-alone, this is needed to wrap them in the context.